mautic/mautic
# Fix contact merge disabled field

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for features or enhancements / 3.2 for bug fixes 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
This PR fix 

`[2021-01-19 15:04:35] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Type error: Argument 2 passed to Mautic\CoreBundle\Helper\ArrayHelper::getValue() must be of the type array, boolean given, called in .../app/bundles/LeadBundle/Deduplicate/ContactMerger.php on line 185"....`



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Disable attributions custom field
3. Create form with email field
4. Create contact 
5. Go to incognito window with form and try submit the form with already created contact
6. Should return error 500
7. After PR should working properly.
